example/winrt_picker.dart

```diff
@@ -17,8 +17,8 @@ void main() async {
       'Windows.Storage.Pickers.FileOpenPicker', IID_IFileOpenPicker);
 
   final picker = IFileOpenPicker.fromRawPointer(object)
-    ..SuggestedStartLocation = PickerLocationId.Desktop
-    ..ViewMode = PickerViewMode.Thumbnail;
+    ..SuggestedStartLocation = PickerLocationId.Desktop.value
```

Comment:
This makes me a little sad ;) because it is a bit of a backward step for the consumer of this package. Is it not possible to keep these as classes with const values? If not, I wonder if we should change the property to take an enum rather than an int...